app.js

```diff
@@ -0,0 +1,107 @@
+//Guessing Game, Three questions
+//Quetions one
+//Do you own an umbrella
+//If yes, You are new to the Seattle area
+//If No, Good real Seattleites don't use umbrallas
+var counter = 0;
+var umbrella = "yes";
+var sunny = "yes";
+var good = "yes";
+var answer = "yes";
+
+
+while(counter < 4){
+var userInput = prompt("Do you own an umbrella?");
+if(umbrella === userInput){
+alert("You are new to the Seattle area");
+console.log("true");
```

Comment:
Console logs are just like comments - they're helpful locally for debugging, but generally shouldn't make it to your production code.